src/github.com/getlantern/flashlight/proxiedsites/proxiedsites.go

```diff
@@ -0,0 +1,70 @@
+package proxiedsites
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"sync"
+
+	"github.com/getlantern/golog"
+	"github.com/getlantern/proxiedsites"
+
+	"github.com/getlantern/flashlight/config"
+	"github.com/getlantern/flashlight/ui"
+)
+
+var (
+	log = golog.LoggerFor("proxiedsites-flashlight")
+
+	uichannel  *ui.UIChannel
+	startMutex sync.Mutex
+)
+
+func Configure(cfg *proxiedsites.Config) {
+	delta := proxiedsites.Configure(cfg)
+	startMutex.Lock()
+	if uichannel == nil {
+		start()
+	} else if delta != nil {
+		b, err := json.Marshal(delta)
+		if err != nil {
+			log.Errorf("Unable to publish delta to UI: %v", err)
+		}
+		uichannel.Out <- b
+	}
+	startMutex.Unlock()
+}
+
+func start() {
+	// Register the PAC handler
+	url := ui.Handle("/proxy_on.pac", http.HandlerFunc(proxiedsites.ServePAC))
```

Comment:
I didn't see any reason to store the pac file on disk, and it added a fair amount of complexity, so now the proxiedsites package just serves up the PAC directly.